.nvmrc

```diff
@@ -1 +1 @@
-20.18
+22.11
```

Comment:
Node v22 causes prettier plugin warnings: 
![image](https://github.com/user-attachments/assets/6c5cb20a-d191-4cdc-a05e-175341a7ab5e)

Prettier v3.4 solves this issue. Could you bump the prettier version?